xtrack/beam_elements/elements.py

```diff
@@ -885,3 +895,99 @@ def __init__(self, length = 0.0, m0 = None, m1 = None,radiation_flag=0,**nargs):
         super().__init__(**nargs)
 
 
+class IonLaserIP(BeamElement):
```

Comment:
I believe this element was added by mistake and should be removed (otherwise tests should be added)